checks.d/process.py

```diff
@@ -328,6 +329,11 @@ def check(self, instance):
             # we use Process(pid) as a means to search, if pid not found
             # psutil.NoSuchProcess is raised.
             pids = set([psutil.Process(pid).pid])
+        elif pid_file is not None:
+            with open(pid_file, 'r') as file_pid:
+                pid_line = file_pid.readline().strip()
+                pids = set([psutil.Process(int(pid_line[-1])).pid])
+            file_pid.close()
```

Comment:
Since you're using a `with` statement you should remove `file_pid.close()` here (python closes the file automatically once we exit the `with` block)